linux_os/guide/services/ntp/ntpd_run_as_ntp_user/rule.yml

```diff
@@ -0,0 +1,40 @@
+documentation_complete: true
+
+prodtype: rhel7,fedora
+
+title: 'Ensure that ntpd is running under ntp user account'
+
+description: |-
+    ntp is a daemon which implements the Network Time Protocol (NTP). It is designed to
+    synchronize system clocks across a variety of systems and use a source that is highly
+    accurate. More information on NTP can be found at
+    {{{ weblink(link="http://www.ntp.org")}}}.
+    ntp can be configured to be a client and/or a server.
+    To ensure that ntpd is running under ntp user account, Add or edit the
+    <tt>OPTIONS</tt> variable in <tt>/etc/sysconfig/ntpd</tt> to include ' -u ntp:ntp ':
+    <pre>OPTIONS="-u ntp:ntp"</pre>
+    This recommendation only applies if ntp is in use on the system.
+
+rationale: |-
+    If ntp is in use on the system proper configuration is vital to ensuring time synchronization
+    is working properly.
```

Comment:
I think a better rationale here is needed for why ntp should run as the ntp user.